terraform/modules/notification_queue/main.tf

```diff
@@ -14,7 +14,7 @@ module "input_queue" {
 
   queue_name = "${var.action_name}-${var.environment}"
 
-  topic_arns                 = [module.notification_topic.arn]
+  topic_arns                 = concat(var.extra_topics, [module.notification_topic.arn])
```

Comment:
It's not super clear why there's a "notification_topic" and also "extra_topics"

Reply:
This notification queue module creates an SNS/SQS pair, so the SQS is fed by the SNS (notification topic).

The Restoration->Transfer transition requires something to happen on one account and result in queue messages on the other.

It seemed to be easier and clearer (as well as the Right Thing to Do, semantically) for the source to notify its own topic, and SQS to listen to that across the account boundary, rather than for the source to notify across the account boundary